src/pages/AboutUsPage/AboutUsPage.scss

```diff
@@ -0,0 +1,73 @@
+
+
+
+.line{
+    background-color: rgb(234, 222, 215);
+    min-height: 20vh;
+    display: flex;
+    align-items: center;
+    
+    h3{
+        font-size: 1.5rem;
+        color: #453227;
+        letter-spacing: 0.1rem;
+        margin-left: 0% !important;
+        
+        a{
+            
+            color: #795744;;
+            padding: 0.5rem;
+            transition: all 0.3s linear;
+            text-decoration: none;
+            &:hover{
+            color: #453227;
+            
+            }
+        }
+    }
+}
+
+
+
+.our-story{
+    padding: 5rem 0;
+    
+    display: grid;
+  gap: 4rem;
+  img {
+    width: 100%;
+    display: block;
+    border-radius: 0.25rem;
+    height: 500px;
+    object-fit: cover;
+  }
+  p {
+    line-height: 2;
+    max-width: 45em;
+    margin: 0 auto;
+    margin-top: 2rem;
+    color: #617d98;
+  }
+  .title {
+    text-align: left;
+  }
+
+  h2{
+    font-size: 2rem;
+    letter-spacing: 0.1rem;
+    margin-bottom: 0.75rem;
+    color: #102a42;
+  }
+  .underline {
+    margin-left: 0;
+    height: 0.25rem;
+    width: 6rem;
+    margin-right: auto;
+    background-color: #ab7a5f;
```

Comment:
please use variables for all the colors in this file

Reply:
okay i will use